### DISCUSSION TITLE
Can't access GitHub when sharing wifi from phone

### DISCUSSION THREAD

**Discussion by lifeBalance** [community/community#21566] [New to GitHub]
<p>I just moved to a flat without internet, so I have to share my connection from my phone using a hotspot. Everything works fine even with other git providers. But for some reason I can’t access any thing on GitHub, not even the main page. From the phone there’s no problem accessing but not from any of my laptops.</p>

**Comment by galaxyeden** [comment]
<p>Hi <a class="mention" href="/lifebalance">@lifeBalance</a> and welcome to the GitHub Support Community! If you’re able to connect from on your phone, I expect this is likely software running on your computer. First, I’d like to check if your computer’s DNS provider allows you to resolve GitHub. Could you please <code>nslookup</code> or <code>dig</code> <a href="http://GitHub.com">GitHub.com</a>?</p>
<p>Next, I’d like to know the exact error message you’re receiving.</p>
<p>Finally, I’d like you to try a test clone using both HTTPS and SSH, to determine if there is something blocking HTTPS traffic to GitHub.</p>
<p>Please let me know what you find. Thanks! Allie</p>

**Comment by lifeBalance** [comment]
<p>Nslookup GitHub gives me:<br>
;; connection timed out; no servers could be reache(I <a href="http://added.com" rel="noopener nofollow ugc">added.com</a> at the end, new users can’t paste links here)<br>
The output of dig github is similar.<br>
I can access <a href="http://gitlab.com" rel="noopener nofollow ugc">gitlab.com</a> without issue, so https seems to work.<br>
How do I do a test clone?</p>

**Comment by galaxyeden** [comment]
<p>Hi <a class="mention" href="/lifebalance">@lifeBalance</a> if you can’t get a DNS lookup reply for GitHub, this points to either a custom DNS server or software on your computer proxying DNS and blocking GitHub - do you have any security software running you can temporarily disable to test it?</p>
<p>Can you please provide the full output from <code>nslookup github.com</code> as well as <code>nslookup sitethatworks.com</code> - replacing <code>sitethatworks.com</code> with anything you can access? Thanks!</p>

**Comment by lifeBalance** [comment]
<p>Well I have the same issue on two machines: one running macos and another on Ubuntu. None have any software that I know off that could interfere.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/d/cde367783940f856a7b37b78941440af9b2b755a.jpeg" data-download-href="/uploads/short-url/tnn5K8lBU8VakcaCwcRe5N4kDJo.jpeg?dl=1" title="IMG20210614124648" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/3051496/181065631-0000677b-bab5-47f0-b76d-e041f346dc05.jpeg" alt="IMG20210614124648" data-base62-sha1="tnn5K8lBU8VakcaCwcRe5N4kDJo" width="666" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/d/cde367783940f856a7b37b78941440af9b2b755a_2_666x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/c/d/cde367783940f856a7b37b78941440af9b2b755a_2_999x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/d/cde367783940f856a7b37b78941440af9b2b755a_2_1332x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/d/cde367783940f856a7b37b78941440af9b2b755a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">IMG20210614124648</span><span class="informations">4000×3000 2.76 MB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment]
<p>That photo tells us that your laptop seems to be using an internal DNS cache.</p>
<p>How about the query for <a href="http://github.com">github.com</a>?</p>

**Comment by lifeBalance** [comment]
<p>Hmm, is it possible to clean that cache?<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/5/151bb0a7ff1258e676ef084b1cc15415db13da62.jpeg" data-download-href="/uploads/short-url/30JlRgw43gKG2gUkhZmKOmU4Afg.jpeg?dl=1" title="IMG20210614133947" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/3051496/181065633-38029a6a-18c4-4d2f-9b9a-820442a68781.jpeg" alt="IMG20210614133947" data-base62-sha1="30JlRgw43gKG2gUkhZmKOmU4Afg" width="666" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/5/151bb0a7ff1258e676ef084b1cc15415db13da62_2_666x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/1/5/151bb0a7ff1258e676ef084b1cc15415db13da62_2_999x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/5/151bb0a7ff1258e676ef084b1cc15415db13da62_2_1332x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/5/151bb0a7ff1258e676ef084b1cc15415db13da62_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">IMG20210614133947</span><span class="informations">4000×3000 3.81 MB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment]
<p>That’s really odd, the cache should at least give you an error response! If you can try <code>dig</code>, that’s usually a little more verbose.</p>
<p>It’s probably possible to clear the cache, how depends on your system. E.g. on an Ubuntu or Debian desktop system you probably either have <code>systemd-resolved</code>, or a <code>dnsmasq</code> instance started by NetworkManager, unless you set up something else. No idea about non-Linux systems.</p>

**Comment by lifeBalance** [comment]
<p>Odd indeed, especially bc the issue appears in 2 different machines and only with <a href="http://GitHub.com" rel="noopener nofollow ugc">GitHub.com</a></p>

**Comment by lifeBalance** [comment, marked as answer]
<p>Ok, found the source of the issue. There’s a setting on my sh**ty android phone named <strong>Private DNS</strong>. I had to <strong>turn it off</strong>, and finally could access github from my laptop connected to my shared WIFi. (Go figure what that setting does…)</p>

**Comment by flutterDeveloperAoeonx** [comment]
same issue even my Private DNS is off ,

i have on DNS again off and restart phone now it's working fine .

thanks ..
